source/extensions/http/original_ip_detection/xff/xff.cc

```diff
@@ -10,16 +11,49 @@ namespace Xff {
 
 XffIPDetection::XffIPDetection(
     const envoy::extensions::http::original_ip_detection::xff::v3::XffConfig& config)
-    : xff_num_trusted_hops_(config.xff_num_trusted_hops()) {}
+    : xff_num_trusted_hops_(config.xff_num_trusted_hops()),
+      skip_xff_append_(PROTOBUF_GET_WRAPPED_OR_DEFAULT(
+          config, skip_xff_append, config.has_xff_trusted_cidrs() ? false : true)) {
+  if (config.has_xff_trusted_cidrs() && config.xff_num_trusted_hops() > 0) {
+    ENVOY_LOG(warn, "Both xff_num_trusted_hops and xff_trusted_cidrs are configured; only "
+                    "xff_trusted_cidrs will be used");
+  }
```

Comment:
It should be safe to reject the configuration?

Reply:
I had previously tried to do that by `throw`ing, but this causes failures for Mobile. Is there another way to reject the configuration?

Reply:
Maybe you can should use `throwExceptionOrPanic` method for now.

Reply:
Is panicking acceptable for Mobile?

Reply:
I've added the `throwExceptionOrPanic`. Hopefully this won't cause the Mobile tests to fail this time. Thanks for the pointer.